account_ux/wizards/account_change_currency.py

```diff
@@ -64,6 +66,7 @@ def change_currency(self):
             self.currency_rate)
 
         move = self.move_id.with_context(check_move_validity=False)
+        move.currency_id = self.currency_to_id.id
```

Comment:
this same line is in both, line 69 and line 73. need to leave only one.
reviewif actually necessary the change